ios/AppLovinMAXNativeAdView.m

```diff
@@ -196,6 +196,9 @@ - (void)setCallToActionView:(NSNumber *)tag
         return;
     }
     
+    // Workaround for the Google adapter to turn this off on its integration
+    view.userInteractionEnabled = YES;
```

Comment:
I think this actually should be fixed SDK-side or adapter-side, since this issue will affect other plugins too, we should try to fix it at the core layer,.. thoughts?